PIL/ImtImagePlugin.py

```diff
@@ -69,7 +69,7 @@ def _open(self):
                 s = s + self.fp.readline()
                 if len(s) == 1 or len(s) > 100:
                     break
-                if s[0] == b"*":
+                if s[0] == ord(b"*"):
```

Comment:
Not a ding against the PR, but the ImtImagePlugin is essentially untested, or at the most generous interpretation, it's tested in that it doesn't open any of the non-imt images that we try to open.